.github/workflows/benchmark-hierarchies.yml

```diff
@@ -7,15 +7,15 @@ on:
       - master
     paths:
       - apps/performance-tests/**
```

Comment:
should this point to `hierarchies` sub-directory?

Reply:
Yes, it should. Fixed.